### PR TITLE
Rental cost solution

### PR DESCRIPTION
**Pull request by yuliia-dronova**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+  let total = 0;
+
+  if (days < 3) {
```
Comment (vitmixG) [inline src/calculateRentalCost.js:36]: The fifth rule of the checklist.

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,20 @@
...
+    total = days * dailyRent;
+  } else if (days >= 3 && days < 7) {
+    total = days * dailyRent - minDiscount;
+  } else {
```
Comment (vitmixG) [inline src/calculateRentalCost.js:40]: The third rule of checklist
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

**Review by vitmixG — CHANGES_REQUESTED** [review]
Good job, but fix a few points

File: src/calculateRentalCost.js (hunk 2 above)
Comment (yuliia-dronova) [reply src/calculateRentalCost.js]: thank you, everything's corrected

**Review by vitmixG — APPROVED** [review]
Well done:)

**Review by solaryasha — APPROVED** [review]
Good job !

**Pull request closed without merging**